<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCIFeatureAddDig</class>
 <widget class="QWidget" name="QCIFeatureAddDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCIFeatureAddDig</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbn_Ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbn_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
